lib/importers/twilio/call_record.rb

```diff
@@ -3,35 +3,95 @@ module Twilio
     class CallRecord
       MINIMUM_CALL_TIME = 10
 
-      def initialize(calls)
+      def params # rubocop:disable Metrics/MethodLength
+        {
+          uid: uid,
+          called_at: called_at,
+          inbound_number: inbound_number,
+          outbound_number: outbound_number,
+          call_duration: outbound_call_duration,
+          cost: cost,
+          outcome: outcome,
+          delivery_partner: @location_details['delivery_partner'],
+          location_uid: @location_details['uid'],
+          location: @location_details['location'],
+          location_postcode: @location_details['location_postcode'],
+          booking_location: @location_details['booking_location'],
+          booking_location_postcode: @location_details['booking_location_postcode']
+        }
+      end
+
+      def initialize(calls, location_details)
         @inbound_call, @outbound_call = *calls
+        @location_details = location_details
+      end
+
+      def uid
+        @inbound_call.parent_call_sid || @inbound_call.sid
+      end
+
+      def called_at
+        Time.zone.parse(@inbound_call.start_time)
       end
 
       def date
-        Time.zone.parse(@inbound_call.start_time).utc.to_date
+        called_at.utc.to_date
+      end
+
+      def inbound_number
+        @inbound_call.to
+      end
+
+      def outbound_number
+        @outbound_call&.to
+      end
+
+      def outbound_call_duration
+        @outbound_call&.duration.to_i
+      end
+
+      def cost
+        @inbound_call.price.to_f + @outbound_call&.price.to_f
```

Comment:
What's the precision on these numbers? Using floats for currency doesn't usually end well.